### PR TITLE
Changes to Exercise and related classes

### PR DESCRIPTION
**Pull request by xingjie99**

### CODE REVIEW

**Review by weidak — COMMENTED** [review]
Looks good to merge!

File: src/main/java/seedu/duke/commands/DeleteExerciseCommand.java [hunk 1]
```diff
@@ -24,6 +25,11 @@ public DeleteExerciseCommand(int itemIndex) {
 
     @Override
     public CommandResult execute() {
```
Comment (tttyyzzz) [inline src/main/java/seedu/duke/commands/DeleteExerciseCommand.java:27]: Nice method

**Review by tttyyzzz — COMMENTED** [review]
Seems good to me, nice method, perhaps updating user guide also. @tryyang2001 *hint*

File: src/main/java/seedu/duke/parser/Parser.java [hunk 2]
```diff
@@ -108,10 +110,19 @@ private Command parseDeleteItems(String params) {
         try {
             final String itemTypePrefix = extractItemTypePrefix(params);
             final String description = extractItemDescription(params, itemTypePrefix).split(" ")[0];
-            final int itemIndex = convertItemNumToItemIndex(Integer.parseInt(description.trim()));
+            final int itemIndex;
+            boolean isClear = description.trim().toLowerCase().equals("all");
```
Comment (tlyi) [inline src/main/java/seedu/duke/parser/Parser.java:114]: Could consider extracting this out as a constant too!

**Pull request merged**